api-reference/10 UI Components/dxChart/1 Configuration/argumentAxis/breaks/breaks.md

```diff
@@ -10,11 +10,11 @@ notUsedInTheme:
 Declares a scale break collection. Applies only if the axis' [type](/api-reference/10%20UI%20Components/dxChart/1%20Configuration/argumentAxis/type.md '/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/#type') is *"continuous"* or *"logarithmic"*.
 
 ---
-A scale break allows breaking off a part of the axis to improve the readability of a chart with high amplitude values.
+A scale break is an area across an axis that is displayed instead of a section of an axis range. It allows you to improve the readability of chart sections with large gaps in their ranges.
```

Comment:
```suggestion
A scale break is an area across an axis that is displayed instead of a section of an axis range. Scale breaks improve the readability of chart sections with large gaps in their ranges.
```

Reply:
Sounds good.